Injects-Incidents-Infrastructure/README.md

```diff
@@ -0,0 +1,109 @@
+# Injects-Incidents-Infrastructure <!-- omit-from-toc -->
+This directory contains materials relating to possible injects, incidents and the infrastructure (it's setup) that we are using. Of note should be the section [Writing Guide](#writing-guide) since this is the main reason this README was made. 
```

Comment:
...are using. of note...  Typo, I would just change it but I don't really know what it means.